src/components/reportPage/ReportPage.jsx

```diff
@@ -51,6 +81,33 @@ const ReportPage = (props) => {
     setSearchString(e.target.value);
   };
 
+  const handleMenuChange = (e) => {
+    getValue(Array.isArray(e) ? e.map((x) => x.value) : []);
+  };
+
+  const checkImageFilterType = (imageResult) => {
+    const found = displayValue.some((value) => imageResult["bug_type"].includes(value));
+
+    if (
+      displayValue.length === 0 ||
+      (displayValue.includes("1") && imageResult["bug_type"].length === 0) ||
```

Comment:
Maybe define "1" as a constant to give it meaning? `const IMAGE_NO_DEFECT_VALUE = "1";`
`displayValue.includes(IMAGE_NO_DEFECT_VALUE)`